public/index.html

```diff
@@ -21,7 +21,7 @@
     </script>
 
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
```

Comment:
we shoudl use ico format so that it looks good in a dark tab
```suggestion
    <link rel="icon" href="%PUBLIC_URL%/favicon.png" />
```
<img width="254" alt="Screenshot 2024-03-25 at 11 20 33 PM" src="https://github.com/cesine/fantastic-lamp/assets/196199/59506072-68cf-444f-904c-285d1f380beb">